### PR TITLE
CodeIgniter 4.0+ auto-instrumentation (root span)

### PR DESCRIPTION
**Pull request by agoallikmaa**
Implements CodeIgniter 4.0+ auto-instrumentation (`open-telemetry/opentelemetry-auto-codeigniter` package). Currently only collects root span.

### CODE REVIEW

**Comment by brettmc** [comment]
Hi @agoallikmaa
Thanks for your contribution, this looks good.
Can you please also update:
- .github/workflows/php.yml (add to build matrix)
- .gitsplit.yaml (to enable git split to a readonly repo that packagist can reference)

File: src/Instrumentation/CodeIgniter/README.md
Comment (brettmc) [inline src/Instrumentation/CodeIgniter/README.md:1]: Can you make this file look more like the other readme's, which were recently all updated?

Comment (agoallikmaa) [reply src/Instrumentation/CodeIgniter/README.md:1]: Added some segments under overview that are present for most of the other autoinstrumentation readmes, hopefully that was what was you meant.

**Comment by brettmc** [comment]
Sorry, I was referring to the changes in https://github.com/open-telemetry/opentelemetry-php-contrib/pull/200 which I thought had already been merged (as discussed in SIG just now)

**Review by brettmc — APPROVED** [review]

**Comment by brettmc** [comment]
https://packagist.org/packages/open-telemetry/opentelemetry-auto-codeigniter

**Pull request merged**